### PR TITLE
Move @uniswap and @ensdomains to "dependencies"

### PR DESCRIPTION
**Pull request by NunoAlexandre**
These are not available in the exported build that clients use, requiring
"all" clients to add these exact dependencies themselves in order to run
the 'deploy.ts' script. That should happen here instead to that it can
actually offer the functionality it proposes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,4 +48,4 @@
     "build/**",
     "contract-bindings/**"
   ]
-}
+}
```
Comment (geigerzaehler) [inline package.json:51]: Could you keep the newline to avoid unnecessary diff noise?

**Review by geigerzaehler — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (NunoAlexandre) [inline package.json:51]:
```suggestion
}

```

Comment (NunoAlexandre) [reply package.json]: That was unintentional, sorry. Will check my editor :eyes:

Fixed btw
